pkg/metallb/frrnodestate_test.go

```diff
@@ -0,0 +1,174 @@
+package metallb
+
+import (
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/metallb/frrtypes"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	defaultNodeName           = "worker-0"
+	defaultFrrNodeStateNsName = "test-namespace"
+)
+
+func TestFrrNodeStateGet(t *testing.T) {
+	testCases := []struct {
```

Comment:
Make sure to have a test case where the object does not exist like in [mco/mcp_test.go](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/mco/mcp_test.go#L144-L147)

Reply:
added